.github/workflows/npm-publish.yaml

```diff
@@ -12,25 +12,77 @@ jobs:
     strategy:
       matrix:
         package:
-          - "@requestnetwork/add-stakeholder"
-          - "@requestnetwork/create-invoice-form"
-          - "@requestnetwork/invoice-dashboard"
-          - "@requestnetwork/shared"
+          - "add-stakeholder"
+          - "create-invoice-form"
+          - "invoice-dashboard"
+          - "shared"
```

Comment:
We must use `package/add-stakeholder` for example because we are checking the local packages, then on in the workflow we use npmjs to check the published packages